test-site/src/components/MapPin.tsx

```diff
@@ -17,20 +16,21 @@ const transformToMapboxCoord = (coordinate: Coordinate): LngLatLike => ({
 });
 
 export const MapPin: PinComponent<Location> = (props: MapPinProps<Location>) => {
-  const { coordinate, mapbox, result } = props;
+  const { mapbox, result } = props;
+  const yextCoordinate = result.rawData.yextDisplayCoordinate;
   const [active, setActive] = useState(false);
   const popupRef = useRef(new Popup({ offset: 15 })
     .on('close', () => setActive(false))
   );
 
   useEffect(() => {
-    if (active) {
+    if (active && yextCoordinate) {
```

Comment:
I'm assuming we need to keep this in because of TypeScript, but is the `yextCoordinate` check actually necessary here? Doesn't the `Mapbox` Component take care of that essentially?

Reply:
Out of curiosity is it possible to make `yextCoordinate` required in the `Location` interface?